src/internal/jsonschema/BUILD.bazel

```diff
@@ -525,6 +525,7 @@ go_library(
         "validate/UInt32Rules.schema.json",
         "validate/UInt64Rules.schema.json",
         "versionpb_v2/Version.schema.json",
+        "logs/JobDatumLogQuery.schema.json",
```

Comment:
I’m guessing this was due to a previous PR not regenerating protos or running gazelle?

Reply:
yup